db/migrate/20220110210841_add_type_to_cdn_configuration.rb

```diff
@@ -0,0 +1,13 @@
+class AddTypeToCdnConfiguration < ActiveRecord::Migration[6.0]
+  def change
+    add_column :katello_cdn_configurations, :type, :string, default: ::Katello::CdnConfiguration::CDN_TYPE
```

Comment:
Having a column called `type` can cause lots of problems:
```
(ActionView::Template::Error): The single-table inheritance mechanism failed to locate the subclass: 'redhat_cdn'. This error is raised because the column 'type' is reserved for storing the class in case of inheritance. Please rename this column if you didn't intend it to be used for storing the inheritance class or overwrite Katello::CdnConfiguration.inheritance_column to use another column for that information.
```
 Let's rename this `cdn_configuration_type` or something.

Reply:
We can keep it as 'type' if you want. I didn't realize I needed to roll back 2 migrations and not just one.  Partha is already overwriting `Katello::CdnConfiguration.inheritance_column` so it shouldn't be an issue for users.